packages/runtime/client/rest/src/contract/endpoints/collection.ts

```diff
@@ -192,12 +199,30 @@ export const GetContractsResponseGuard = assertGuardEqual(
 );
 
 /**
- * Request options for the {@link index.RestClient#createContract | Create contract } endpoint
- * @category Endpoints
+ * Either a Non Merkleized Marlowe Contract or a Merkleized One
+ * @category Endpoint : Build Create Contract Tx
+ */
+export type ContractOrSourceId = Contract | SourceId;
+
+/**
+ * Guard for ContractOrSourceId type
```

Comment:
Notice that both in the core-v1 and marlowe-objects package I document all guards [this way](https://github.com/input-output-hk/marlowe-ts-sdk/blob/main/packages/language/core/v1/src/payee.ts#L41) That allows me to eventually change All guards documentation by a simple search and replace if we decide that there is a better message.

Reply:
https://github.com/input-output-hk/marlowe-ts-sdk/issues/118